src/Aquifer.API/Endpoints/Resources/Content/NeedsTranslation/List/Validator.cs

```diff
@@ -0,0 +1,40 @@
+using FastEndpoints;
+using FluentValidation;
+
+namespace Aquifer.API.Endpoints.Resources.Content.NeedsTranslation.List;
+
+public class Validator : Validator<Request>
+{
+    public Validator()
+    {
+        RuleFor(x => x.ParentResourceId)
+            .GreaterThan(0).When(x => x.ParentResourceId.HasValue);
+
+        RuleFor(x => x.BookCode)
+            .NotEmpty().When(x => x.StartChapter.HasValue && x.EndChapter.HasValue)
+            .WithMessage("BookCode must be specified if StartChapter and EndChapter are specified.");
+
+        RuleFor(x => x.StartChapter)
+            .GreaterThan(0).When(x => x.StartChapter.HasValue);
```

Comment:
I was messing around with validation on nullable properties and IIRC the `When(x => x.Foo.HasValue)` is not necessary.  The validator's smart enough to only do the greater than check if the value is non-null.

Reply:
Ah good find, this was copypasta from a different endpoint but would be good to update that in both spots at some point

Reply:
Yeah, if want to verify something isn't null have to add the `NotEmpty()` on there too. Feels weird, but seems to work best most of the time.